common/src/main/java/tachyon/security/authorization/FileSystemPermission.java

```diff
@@ -67,6 +67,14 @@ public FileSystemAction getUserAction() {
   }
 
   /**
+   * @param permission the digital representation of a {@link FileSystemPermission}
```

Comment:
better to explain it. for example, you can simple add "(e.g., 777 for rwxrwxrwx)" after "digital representation"